src/features/completions.zig

```diff
@@ -876,7 +876,7 @@ pub fn completionAtIndex(
 
     switch (pos_context) {
         .builtin => try completeBuiltin(&builder),
-        .var_access, .empty => try completeGlobal(&builder),
+        .var_access, .empty, .parens_expr => try completeGlobal(&builder),
```

Comment:
According to the following [comment](https://github.com/zigtools/zls/blob/18c1c3947d52f6db55118731621d724a799481da/src/analysis.zig#L3282), `parens_expr` is only supposed to be used internally.

`parens_expr` should be converted to `var_access` [here](https://github.com/zigtools/zls/blob/18c1c3947d52f6db55118731621d724a799481da/src/analysis.zig#L3557-L3565).